src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -34,46 +34,39 @@
         </core:EventTriggerBehavior>
     </interactivity:Interaction.Behaviors>
 
-    <Grid>
+    <Grid RowSpacing="8">
         <Grid.RowDefinitions>
-            <RowDefinition Height="48" />
+            <RowDefinition Height="Auto" x:Name="ButtonsRow" />
```

Comment:
There needs to be more spacing between the caption buttons and the 'command' buttons (see left image)

<img width="998" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/2794277d-596e-4b31-be64-d153033f1a82">

Reply:
8 will do?